### ISSUE TITLE
Missing fields to be tracked

### ISSUE THREAD

**Issue by WHITE-developer** [putyourlightson/craft-blitz#682]
### Bug Report

While debugging why my page doesn't get refreshed I found that some of my custom (renamed) fields aren't been tracked. The situation:

1. Entry with matrix in card view
2. Block with for example a custom title field. Custom handle `blockTitle` , original handle `text`
3. Field is for sure outputted on the page
4. Saving that block after changing the field should trigger a refresh for the page

Tried refreshing en flushing the cache and then revisiting the page.

I can imagine that it is hard to debug with this, so I could sent you a DB dump if you like.

#### Application Info

- PHP version: 8.3.7
- Craft edition & version: Pro 5.2.5
- Database driver & version: MySQL 8.0.37

#### Installed Plugins

- Amazon S3: 2.2.1
- Blitz: 5.4.0
- CKEditor: 4.1.0
- Embedded Assets: 5.1.0
- Formie: 3.0.0-beta.16
- Hyper: 2.0.2
- Imager X: 5.0.1
- Navigation: 3.0.1
- Retour: 5.0.2
- SEOmatic: 5.0.4
- Sprig: 3.0.2
- Table Maker: 5.0.0

#### Loaded Modules

- backend: modules\backend\BackendModule
- connector: modules\connector\ConnectorModule
- frontend: modules\frontend\FrontendModule
- imports: modules\imports\Module
- codeeditor: nystudio107\codeeditor\CodeEditor
- sprig-core: putyourlightson\sprig\Sprig
- verbb-base: verbb\base\Base

#### Blitz Plugin Settings

```json
{
    "debug": false,
    "hintsEnabled": true,
    "cachingEnabled": true,
    "refreshCacheEnabled": true,
    "refreshMode": 1,
    "includedUriPatterns": [
        {
            "siteId": "",
            "uriPattern": ".*"
        }
    ],
    "excludedUriPatterns": [],
    "cacheStorageType": "putyourlightson\\blitz\\drivers\\storage\\FileStorage",
    "cacheStorageSettings": {
        "folderPath": "@webroot\/cache\/blitz",
        "compressCachedValues": ""
    },
    "cacheStorageTypes": [],
    "cacheGeneratorType": "putyourlightson\\blitz\\drivers\\generators\\HttpGenerator",
    "cacheGeneratorSettings": {
        "concurrency": 3
    },
    "cacheGeneratorTypes": [],
    "customSiteUris": [],
    "cachePurgerType": "putyourlightson\\blitz\\drivers\\purgers\\DummyPurger",
    "cachePurgerSettings": [],
    "cachePurgerTypes": [],
    "deployerType": "putyourlightson\\blitz\\drivers\\deployers\\DummyDeployer",
    "deployerSettings": [],
    "deployerTypes": [],
    "ssiEnabled": true,
    "ssiTagFormat": "<!--#include virtual=\"{uri}\" -->",
    "detectSsiEnabled": true,
    "esiEnabled": false,
    "queryStringCaching": 0,
    "includedQueryStringParams": [
        {
            "siteId": "",
            "queryStringParam": ".*"
        }
    ],
    "excludedQueryStringParams": [
        {
            "siteId": "",
            "queryStringParam": "gclid"
        },
        {
            "siteId": "",
            "queryStringParam": "fbclid"
        }
    ],
    "apiKey": "",
    "generatePagesWithQueryStringParams": true,
    "purgeAssetImagesWhenChanged": true,
    "refreshCacheAutomaticallyForGlobals": true,
    "refreshCacheWhenElementMovedInStructure": true,
    "refreshCacheWhenElementSavedUnchanged": false,
    "refreshCacheWhenElementSavedNotLive": false,
    "cacheNonHtmlResponses": false,
    "trackElements": true,
    "trackElementQueries": true,
    "excludedTrackedElementQueryParams": [],
    "cacheDuration": 604800,
    "nonCacheableElementTypes": [
        "craft\\elements\\user"
    ],
    "sourceIdAttributes": [],
    "liveStatuses": [],
    "integrations": [
        "putyourlightson\\blitz\\drivers\\integrations\\SeomaticIntegration"
    ],
    "defaultCacheControlHeader": "no-cache, no-store, must-revalidate",
    "cacheControlHeader": "public, s-maxage=604800, max-age=0",
    "cacheControlHeaderExpired": "public, s-maxage=5, max-age=0",
    "sendPoweredByHeader": false,
    "outputComments": true,
    "refreshCacheJobPriority": 110,
    "driverJobBatchSize": 100,
    "driverJobPriority": 100,
    "queueJobTtr": 3600,
    "maxRetryAttempts": 10,
    "maxUriLength": 255,
    "mutexTimeout": 1,
    "commands": [],
    "injectScriptEvent": "DOMContentLoaded",
    "injectScriptPosition": 3
}
```

#### Recommendations

- ❌ The Blitz Hints utility is reporting 1 eager-loading opportunity.
- ❌ The blitz/cache/refresh-expired console command has not been executed within the past 24 hours.
- ✅ Blitz is configured not to refresh cached pages when an element is saved but unchanged.
- ✅ Blitz is configured not to refresh cached pages when an element is saved but not live.
- ✅ Image transforms are configured to be generated before page load.
- ✅ No globals exist.
- ✅ The @web alias is explicitly defined.
- ✅ Queue jobs are configured not to run automatically via web requests.
- ✅ The Async Queue plugin is not installed or enabled.

**Comment by bencroker** [comment]
Confirmed that this indeed is not working and will address in the next release. Thanks for reporting!

**Comment by bencroker** [comment]
Added the ability for Blitz to track custom field instances with renamed handles in https://github.com/putyourlightson/craft-blitz/commit/7028c91b85c393571e84da1e513751e032e407e0.

You can test it by running `composer require "putyourlightson/craft-blitz:dev-develop as 5.5.0"`.

To ensure the changes are applied, the cache should be cleared or refreshed after the update completes.

**Comment by WHITE-developer** [comment]
Looks like it's working :tada: !

One small thing, but not that important (for me): if you don't use the title field in a block, the "Entry" is empty in the diagnostics: 
![image](https://github.com/user-attachments/assets/b2eef965-ab28-4a7a-811d-1b11e9641067)

**Comment by bencroker** [comment]
Thanks, I’ll look into that.

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-blitz/commit/a822039b38ece7a8433c38bcc9e7f125d57b10ec.

**Comment by WHITE-developer** [comment]
Yes, that's better :slightly_smiling_face:. Any idea when you will release it?

**Comment by bencroker** [comment]
Great! It will ship in 5.5.0, likely early next week.

**Comment by bencroker** [comment]
Released in [5.5.0](https://github.com/putyourlightson/craft-blitz/releases/tag/5.5.0).
